From be1d27101036bb8b68457008a657e53c508be1b5 Mon Sep 17 00:00:00 2001
From: Dan Nica <dan.nica@nxp.com>
Date: Wed, 17 Aug 2022 11:31:25 +0300
Subject: [PATCH 074/102] s32: env: Update 'fdt_enable_hs400es'

The name of the uSDHC node in Linux has changed from 'usdhc' to
'mmc', therefore this command must be updated.

Issue: ALB-9092
Upstream-Status: Pending 

Signed-off-by: Dan Nica <dan.nica@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32-cc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32-cc.h b/include/configs/s32-cc.h
index b709d85150..5b5014a5a2 100644
--- a/include/configs/s32-cc.h
+++ b/include/configs/s32-cc.h
@@ -134,7 +134,7 @@
 	"fdt_addr=" __stringify(S32CC_FDT_ADDR) "\0" \
 	"fdt_enable_hs400es=" \
 		"fdt addr ${fdt_addr}; " \
-		"fdt rm /soc/usdhc no-1-8-v; " \
+		"fdt rm /soc/mmc no-1-8-v; " \
 		"fdt resize; \0" \
 	"fdt_file=" FDT_FILE "\0" \
 	"fdt_high=" __stringify(S32CC_FDT_HIGH_ADDR) "\0" \
-- 
2.17.1

